src/DotNetWorker.Core/FunctionMetadata/SkipDefaultFunctionMetadataProviderAggregator.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Core.FunctionMetadata;
+
+public class SkipDefaultFunctionMetadataProviderAggregator : IFunctionMetadataProviderAggregator
```

Comment:
Made public so that the corresponding aggregator can be registered by the extensions